app-maven-plugin/pom.xml

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
This module depends on appengine-plugins-core. Release Please should update that line too.

```
  <dependencies>
    <dependency>
      <groupId>com.google.cloud.tools</groupId>
      <artifactId>appengine-plugins-core</artifactId>
      <version>0.10.1-SNAPSHOT</version>
```

Reply:
Ah, good catch, thanks.